src/Commands/DrushCmiToolsCommands.php

```diff
@@ -0,0 +1,308 @@
+   *
+   * @var \Drush\Drupal\Commands\config\ConfigImportCommands
+   */
+  protected $configImportCommands;
+
+  /**
+   * The config storage.
+   *
+   * @var \Drupal\Core\Config\StorageInterface
+   */
+  protected $configStorage;
+
+  /**
+   * The config manager.
+   *
+   * @var \Drupal\Core\Config\ConfigManagerInterface
+   */
+  protected $configManager;
+
+  /**
+   * CmiToolsCli constructor.
+   *
+   * @param \Drupal\Core\File\FileSystemInterface $fileSystem
+   *   The file system service.
+   * @param \Drupal\Component\Serialization\SerializationInterface $serializer
+   *   The serializer.
+   * @param \Drush\Drupal\Commands\config\ConfigCommands $configCommands
+   *   The config commands.
+   * @param \Drush\Drupal\Commands\config\ConfigExportCommands $configExportCommands
+   *   The config export commands.
+   * @param \Drush\Drupal\Commands\config\ConfigImportCommands $configImportCommands
+   *   The config import commands.
+   * @param \Drupal\Core\Config\StorageInterface $configStorage
+   *   The config storage.
+   * @param \Drupal\Core\Config\ConfigManagerInterface $configManager
+   *   The config manager.
+   */
+  public function __construct(FileSystemInterface $fileSystem, SerializationInterface $serializer, ConfigCommands $configCommands, ConfigExportCommands $configExportCommands, ConfigImportCommands $configImportCommands, StorageInterface $configStorage, ConfigManagerInterface $configManager) {
+    $this->fileSystem = $fileSystem;
+    $this->serializer = $serializer;
+    $this->configCommands = $configCommands;
+    $this->configExportCommands = $configExportCommands;
+    $this->configImportCommands = $configImportCommands;
+    $this->configStorage = $configStorage;
+    $this->configManager = $configManager;
+  }
+
+  /**
+   * Export configuration to a directory and apply an ignore list.
+   *
+   * @param array $options
+   *   The options.
+   *
+   * @command config-export-plus
+   * @option destination An arbitrary directory that should receive the
+   *   exported files. An alternative to label argument.
+   * @option ignore-list Path to YAML file containing config to ignore from
+   *   exports
+   * @usage drush config-export-plus --destination=/some/folder
+   *   --ignore-list=./config-ignore.yml Export configuration; Save files in a
+   *   backup directory named config-export.
+   * @aliases cexy
+   *
+   * @return mixed
+   *   The result.
+   */
+  public function exportPlus(array $options = [
+    'destination' => NULL,
+    'ignore-list' => NULL,
+  ]) {
+    $this->logger()->debug('Starting Exporting.');
+    // Do the actual config export operation
+    // Determine which target directory to use.
+    if ($target = $options['destination']) {
+      $destination_dir = $target;
+      // It is important to be able to specify a destination directory that
+      // does not exist yet, for exporting on remote systems.
+      $this->fileSystem->mkdir($destination_dir, NULL, TRUE);
```

Comment:
This is throwing a warning when the directory exists `mkdir(): File exists FileSystem.php:237`